net/upnp/src/etc/inc/plugins.inc.d/miniupnpd.inc

```diff
@@ -198,7 +198,10 @@ function miniupnpd_configure_do($verbose = false)
                 $config_text .= "bitrate_up={$upload}\n";
             }
 
-            $config_text .= "secure_mode=yes\n";
```

Comment:
I would prefer having else branch still setting this. We don't know when defaults change and it doesn't really hurt either

Reply:
In either case either we do it or I have you on record stating that avoiding the "how could xyz security issue have happened"? Just a review procedure so thanks for bearing with me. 👍